exporter/awsemfexporter/grouped_metric_test.go

```diff
@@ -204,18 +204,22 @@ func TestAddToGroupedMetric(t *testing.T) {
 			generateTestSumMetric("int-sum", intValueType),
 			generateTestSumMetric("double-sum", doubleValueType),
 			generateTestSummaryMetric("summary"),
-			// We do not expect these to be added to the grouped metric. Metrics with NaN values should be dropped.
+			// We do not expect these to be added to the grouped metric. Metrics with NaN or Inf values should be dropped.
 			generateTestGaugeMetricNaN("double-gauge-nan"),
 			generateTestExponentialHistogramMetricWithNaNs("expo-with-nan"),
 			generateTestHistogramMetricWithNaNs("histo-with-nan"),
 			generateTestSummaryMetricWithNaN("sum-with-nan"),
+			generateTestGaugeMetricInf("double-gauge-inf"),
+			generateTestExponentialHistogramMetricWithInfs("expo-with-inf"),
+			generateTestHistogramMetricWithInfs("histo-with-inf"),
+			generateTestSummaryMetricWithInf("sum-with-inf"),
 		}
 
 		finalOtelMetrics := generateOtelTestMetrics(generateMetrics...)
 		rms := finalOtelMetrics.ResourceMetrics()
 		ilms := rms.At(0).ScopeMetrics()
 		metrics := ilms.At(0).Metrics()
-		require.Equal(t, 14, metrics.Len(), "mock metric creation failed")
+		require.Equal(t, 19, metrics.Len(), "mock metric creation failed")
```

Comment:
This requirement just verifies that all of our test metrics are generated right? This require should contain ALL metrics, even nan and inf ones? Is this correct? I remember scratching my head at this assertion last time I worked on this.

Reply:
yes thats my understanding as well.

Reply:
can you add a comment saying as such?? It would be greatly appreciated

Reply:
added a comment, please let me know if there are any other changes